### PR TITLE
Add lecture 2

### PR DESCRIPTION
**Pull request by AthletiCoder**

### CODE REVIEW

File: Lecture2/Code/data_types/string/string.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+sample_string = "Please be ready @ 5 pm!"
+print(sample_string)
+
+# Splicing
```
Comment (AthletiCoder) [inline Lecture2/Code/data_types/string/string.py:4]: slicing?

**Review by AthletiCoder — COMMENTED** [review]
one comment included

File: Lecture2/Code/data_types/string/string.py (hunk 1 above)
Comment (HarshBansal97) [reply Lecture2/Code/data_types/string/string.py:4]: Oh yes, didn't notice. Slicing seems better word here, splice means to join. It can be modified.

Comment (AthletiCoder) [reply Lecture2/Code/data_types/string/string.py:4]: once the changes are done you can only approve the pull request pr. I added a rule that there should be 2 approvers

**Pull request merged**